Cython/Utility/MemoryView_C.c

```diff
@@ -24,6 +24,9 @@ typedef struct {
 #ifndef CYTHON_ATOMICS
     #define CYTHON_ATOMICS 1
 #endif
+// using CYTHON_ATOMICS as a cdef extern bint in the Cython memoryview code
+// interacts badly with "import *". Therefore, define a helper function-like macro
+#define __PYX_CYTHON_ATOMICS_ENABLED() CYTHON_ATOMICS
```

Comment:
Can we move this #define into a verbatim code section in `MemoryView.pyx`? (Assuming that works…) It really only exists to make up for a some problematic behaviour over there – which might even go away at some point.

Reply:
Lovely. Let's keep it as it is then.